test/test_library/functional/test_library.py

```diff
@@ -147,18 +147,19 @@ def test_database_query(self):
         for ref, vid in tests.items():
             ip, results = r.resolve_ref_all(ref)
 
-            self.assertEqual(1, len(results))
+            self.assertEquals(1, len(results))
 
             first = results.values().pop(0)
             vid2 = first.vid if not first.partitions else first.partitions.values()[0].vid
 
             self.assertEquals(vid, vid2)
 
+    @unittest.skip(
```

Comment:
This is a test problem. This number of columns actually changes a lot, every time a column is added to the schema. Maybe it would be better to count a specific type of column ( for instance, with an unusual datatype ) or a rarely used extra field in the column table, such as universe or units